tango_simlib/sim_xmi_parser.py

```diff
@@ -179,6 +179,18 @@ def __init__(self):
         }]
 
         """
+        self.class_description = []
+        """Data structure format is a list containing the Tango class description
+        information in a dict.
+
+        e.g.
+        [{
+           "super_class_num": {
+               "parent_class_name": "",
+               "relative_path_to_parent_xmi_file": ""
```

Comment:
In the .xmi files I see slightly different attribute names in the <inheritances> section: `classname` and `sourcePath`.  The latter seems to be used as an absolute path in the few example I looked at.

Either this docstring needs to change, or `extract_device_class_descr()` needs to change to get the data into the keys promised here.